walkingastro.lic

```diff
@@ -76,8 +76,18 @@ class WalkingAstro
     waitrt?
   end
 
+  def can_see_sky?
+    # If we didn't see the sky last time we checked
+    # and we're in the same room still then don't redundantly check
+    # so that we don't spam the game window with "That's a bit hard to do while inside".
+    return false if @last_room_check == DRRoom.title && @last_sky_check == false
```

Comment:
If we're still in the same room and the last sky check was true, is there a reason to check the weather again?

I.e. could this line instead be `return @last_sky_check if @last_room_check == DRRoom.title`?

Reply:
Great point, yes!

Reply:
Resolved in commit https://github.com/rpherbig/dr-scripts/pull/4451/commits/43594821a601f6cd3f371d2f89e58a8d0d525a91